### PR TITLE
Users import script

### PR DESCRIPTION
**Pull request by jaleszek**
Importing script for users migration from csv file.

### CODE REVIEW

File: app/models/users_import/builder.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+class UsersImport::Builder
+  attr_reader :csv
+
+  def initialize(filepath)
+    file = File.open(filepath).read
+    @csv = CSV.parse(file, headers: true)
+  end
+
+  def process
```
Comment (matthewford) [inline app/models/users_import/builder.rb]: Assignment Branch Condition size for process is too high. [18.47/15]

**Comment by jaleszek** [comment]
@RuslanHamidullin take a look please

**Comment by RuslanHamidullin** [comment]
sure)

**Comment by RuslanHamidullin** [comment]
Looks good :+1:

**Pull request merged**